deepchem/data/tests/test_datasets.py

```diff
@@ -395,6 +397,109 @@ def test_disk_iterate_batch_size(self):
       batch_sizes.append(len(X))
     self.assertEqual([3, 3, 3, 1], batch_sizes)
 
+  def test_disk_iterate_batch(self):
+
+    all_batch_sizes = [
```

Comment:
`all_batch_sizes` gets created, but then it doesn't look like it ever gets used.